QUARKUS-5613.md

```diff
@@ -0,0 +1,83 @@
+## Links to the resources
+- JIRA ticket: https://issues.redhat.com/browse/QUARKUS-5613
+- GH PRs related to the JIRA issue: https://github.com/quarkusio/quarkus/pull/43283 https://github.com/quarkusio/quarkus/issues/42117
+- TP/TD PR: https://github.com/quarkus-qe/quarkus-test-plans/pull/193
+- Guides modified and related:
+  * https://quarkus.io/guides/security-openid-connect-client-reference
+  * https://quarkus.io/guides/security-oidc-code-flow-authentication
+  * https://quarkus.io/guides/security-openid-connect-client-registration
+
+## Scope of the testing
+
+Our main testing scope is to ensure that the new OIDC response filter mechanism works as intended for various OIDC endpoints, ensuring that is possible intercept and handle responses
+from various OIDC endpoints. This complements the already available OidcRequestFilter functionality.
+This kind of validation is possible by asserting that:
+    - response body can be read and serialized,
+    - status code is correct,
+    - response headers are correct
```

Comment:
first 2 list items are ended with a comma and last 2 items are not. I have no idea what is common in English world, but I'd do it stuff in uniform pattern. so let's add command here as well and behind the last one dot, or no commas, thanks